models/articles.models.js

```diff
@@ -18,14 +18,17 @@ exports.selectArticle = async (id) => {
     return article;
 }
 
-exports.selectAllArticles = async () => {
+exports.selectAllArticles = async (sortBy = 'created_at', order = 'desc', topic = '%%') => {
+
+    await this._validateInput({sortBy, order});
     
     const query = `SELECT articles.*, COUNT(comment_id)::INTEGER AS comment_count FROM articles
                     LEFT JOIN comments ON comments.article_id = articles.article_id
+                    WHERE topic ILIKE $1
```

Comment:
this where should be conditional

Reply:
oh my bad, just realised what you have done so ignore previous comment. All good